src/components/screens/timebank-screen.tsx

```diff
@@ -30,28 +30,45 @@ const TimebankScreen = () => {
   const [personTotalTime, setPersonTotalTime] = useAtom(personTotalTimeAtom);
   const [personDailyEntry, setPersonDailyEntry] = useAtom(personDailyEntryAtom);
   const [dailyEntries, setDailyEntries] = useAtom(dailyEntriesAtom);
+  const [selectedEmployee] = useState<number | null>(
+    userProfile?.id ? Number(localStorage.getItem("selectedEmployee") || userProfile.id) : null
+  );
 
   useEffect(() => {
-    getPersonTotalTime();
-  }, [persons, timespan]);
+    if (selectedEmployee !== null) {
```

Comment:
Move this conditional into `getPersonTotalTime()` function

Reply:
Conditional is now moved to getPersonalTotalTime() function.